src/Rule/In.php

```diff
@@ -28,30 +30,33 @@ final class In implements RuleWithOptionsInterface, SkipOnErrorInterface, WhenIn
     use SkipOnErrorTrait;
     use WhenTrait;
 
+    /**
+     * @param iterable $values A set of values to check against.
+     * @psalm-param iterable<scalar> $values
+     *
+     * @param bool $strict Whether the comparison is strict (both type and value must be the same).
```

Comment:
I'd complement this with `Boolean::$strict` info (more detailed explanation, links to PHP docs).